### PR TITLE
Palindrome: Take One

### PR DESCRIPTION
**Pull request by efuller**
## Requirements

- [x] commit on every single transition from red to green to refactor
- [x] parameterize tests
- [x] "mom" returns true
- [x] "Mom" returns true
- [x] "MoM" returns true
- [x] "Momx" returns false
- [x] "xMomx" returns true
- [x] "Was It A Rat I Saw" returns true
- [x] "Never Odd or Even" returns true
- [x] "Never Odd or Even1" returns false 
- [x] "1Never Odd or Even1" returns true

### CODE REVIEW

**Comment by taha-crafter** [comment]
Hi @efuller 

very well done! I see how you followed the Fake it strategy until 3 repetitions appeared, you also have covered all the grading checklist tests.

a very tiny comment below 👇🏼

File: 1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts [hunk 1]
```diff
@@ -5,8 +5,16 @@ describe('palindrome checker', () => {
...
-    expect(palindromeChecker('mom')).toBeTruthy();
+  it.each([
+    'mom',
+    'Mom',
+    'Was It A Rat I Saw',
+    'MoM',
+    'xMomx',
+    'Never Odd or Even',
+    '1Never Odd or Even1'
+  ])('when given %s should return true', (param: string) => {
```
Comment (taha-crafter) [inline 1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts:16]: I'd name these parameterized test something like:
`('knows that %s is a palindrome', (param: string) => {`

File: 1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts [hunk 2]
```diff
@@ -17,23 +17,13 @@ describe('palindrome checker', () => {
...
-
-  it('should know that Never Odd or Even1 is not a palindrome', () => {
-    expect(palindromeChecker('Never Odd or Even1')).toBeFalsy();
+  it.each([
+    'john',
+    'jeff',
+    'lucy',
+    'Momx',
+    'Never Odd or Even1',
+  ])('when given %s should return false', (param: string) => {
```
Comment (taha-crafter) [inline 1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts:26]: and here
``('knows that %s is NOT a palindrome', (param: string) => {``

File: 1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts (hunk 1 above)
Comment (efuller) [reply 1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts:16]: Ah yes, great spot! I will find that lesson again and re-read!

**Pull request merged**